sdk/storage/azure-storage-blob/azure/storage/blob/_lease.py

```diff
@@ -121,9 +105,9 @@ def acquire(self, lease_duration=-1, **kwargs):
                 **kwargs)
         except StorageErrorException as error:
             process_storage_error(error)
-        self.id = response.get('lease_id')  # type: str
-        self.last_modified = response.get('last_modified')   # type: datetime
-        self.etag = kwargs.get('etag')  # type: str
+        self.id = response.get('lease_id')
+        self.last_modified = response.get('last_modified')
+        self.etag = kwargs.get('etag')
```

Comment:
would you like to make it `self.etag = response.get('etag')`, I feel it's a 'typo' in the existing code